src/XMLimport.cpp

```diff
@@ -227,7 +227,7 @@ bool XMLimport::importPackage(QFile* pfile, QString packName, int moduleFlag, QS
         }
     }
 
-    return !error();
+    return {!hasError(), errorString()};
```

Comment:
Did not find where the two new functions are coming from. They are not part of XMLimport.cpp or .h files at least.

Reply:
Thanks! I figured it will probably be some Qt built-in but did not have the time to dig through them, yet.